src/staticList.js

```diff
@@ -0,0 +1,190 @@
+import fetch from "node-fetch";
+import fs from "fs";
+import path from "path";
+import _ from "lodash";
```

Comment:
I know this is a script so it doesn't matter much, but still, can you import only what's needed?
If not for performance, it makes is more readable.
Only used once `_.uniqBy`